### PR TITLE
STROKE PREDICTION

### PR DESCRIPTION
**Pull request by RAJharsh02**
# Pull Request for DL-Simplified 💡

## Issue Title :  Stroke Prediction #290 

- **Info about the related issue (Aim of the project)** : "Developing a precise stroke prediction model using dl method to identify individuals at high risk of stroke for early intervention." 
- **Name:** Harsh Raj 
- **GitHub ID:** https://github.com/RAJharsh02
- **Email ID:**  harshraj2828@gmail.com 
- **Idenitfy yourself: (Mention in which program you are contributing in. Eg. For a JWOC 2022 participant it's, `JWOC Participant`)**   ssoc23 

Closes: #290 

### Describe the add-ons or changes you've made 📃

Implemented an artificial neural network (ANN), Recurrent Neural Network (RNN), and  Feedforward Neural Network (FNN)model for stroke prediction.
Added data preprocessing and feature engineering techniques to improve model performance.
Conducted exploratory data analysis (EDA) to gain insights into the dataset.
Fine-tuned hyperparameters to optimize the model's accuracy and robustness.
Updated the README file with detailed instructions and documentation.

## Type of change ☑️

What sort of change have you made:

- [x]  Bug fix (non-breaking change which fixes an issue)
- [x]  New feature (non-breaking change which adds functionality)
- [x]  Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️

I have splited the input in two sets test and train.
then predict the output using inbuild functions
then verified with the help of inbuild function(evaluate) to get the accuracy

## Checklist: ☑️

- [x]  My code follows the guidelines of this project.
- [x]  I have performed a self-review of my own code.
- [x]  I have commented my code, particularly wherever it was hard to understand.
- [x]  I have made corresponding changes to the documentation.
- [x]  My changes generate no new warnings.
- [x]  I have added things that prove my fix is effective or that my feature works.
- [x]  Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — APPROVED** [review]
Your PR is approved and ready to be merged.
@RAJharsh02
